enhancements/windows-containers/disconnected-support.md

```diff
@@ -0,0 +1,355 @@
+
+When deconfiguring Windows instances, mirror settings will be cleared from the node. This involves undoing some node
+config steps, which will be taken care of as part of removing the `containerd` Windows service.
+This scenario will occur when upgrading both BYOH and Machine-backed Windows nodes.
+
+Downgrades are generally [not supported by OLM](https://github.com/operator-framework/operator-lifecycle-manager/issues/1177),
+which manages WMCO. In case of breaking changes, please see the
+[WMCO Upgrades](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/windows-machine-config-operator-upgrades.md#risks-and-mitigations)
+enhancement document for guidance.
+
+### Version Skew Strategy
+
+N/A. There will be no version skew since this work is all within 1 product sub-component (WMCO). 
+The official Red Hat operator in OCP 4.15, WMCO version 10.15.0, will support Windows containers in disconnected
+environments. Releases 9.y.z and below will not have support disconnected support.
+
+## Implementation History
+
+The implementation history can be tracked by following the associated work items in Jira and source code improvements in
+the WMCO Github repo.
+
+## Alternatives & Justification
+ 
+For disconnected support, we could make the kubelet pause image location configurable. This is a nice to have, but may 
+be out of scope of disconnected support as it could require a larger undertaking to make the containerd config file
+configurable.  A valid workaround exists, which is instructing users to set a mirror (through IDMS/ITMSes) for the MSFT
+external registry where `mcr.microsoft.com/oss/kubernetes/pause:3.9` lives.
+
+There are a few alternatives that are no longer viable.
+Pre-pulling images, where user creates a Windows image with the pause image and other required images baked in, is not
+possible anymore as we install and manage the container runtime.
+Also, users cannot easily configure containerd's registries manually since WMCO will overwrite changes to the containerd
+config file if there is some unrelated event that triggers node reconciliation.
+
+## Open Questions
+
+* Enabling pulling from private registries that require authentication credentials or additional certificates?
+  * Would additional auth like image pull secrets, creds, CAs, etc be usable out of the box through the Internal Registry Operator?
+  “When referencing the Windows image in deployments or pods, use the repository and version tag that you specified while tagging the image
+  OpenShift will handle the appropriate image pulling and deployment for Windows-based workloads (e.g. image pull secrets). 
+  Image Registry Operator should be sufficient to handle image pull config just as it does for Linux images.”
+
+* Would it work "out-of-the-box" today if images were pre-loaded into the OpenShift internal registry,
```

Comment:
Do pre-loaded images need specific tags so that WMCO and workloads can pull them?

Reply:
yes, the information on this is below